bin/check-smart.rb

```diff
@@ -65,8 +65,8 @@ def smart_capable?
   #
   def check_smart_capability!
     output = `sudo smartctl -i #{@device_path}`
-    @smart_available = !output.scan(/SMART support is: Available/).empty?
-    @smart_enabled = !output.scan(/SMART support is: Enabled/).empty?
+    @smart_available = !output.scan(/SMART support is:\s*Available/).empty?
```

Comment:
is this only applicable to whitespace? From the output provided it looks like that is the case. If so lets replace `*` with `+` to make it only look for repeating whitespace.